modules/deployment/src/main/java/org/mule/runtime/module/deployment/internal/FlowStoppedDeploymentPersistenceListener.java

```diff
@@ -24,6 +24,8 @@
 
 /**
  * Defines a listener to persist stop events of Mule flows using deployment properties.
+ *
+ * @since 4.2
```

Comment:
@since 4.4.0 4.3.1 4.2.3 (where this will be cherrypicked, the next patch release)